### ISSUE TITLE
Not loading 'centre' when editing

### ISSUE THREAD

**Issue by Myster** [ArnoldV/Our.Umbraco.GMaps#88]
If I set my map to look like this:
![image](https://user-images.githubusercontent.com/354777/152252330-c58e9726-7ac5-4e22-a079-c777dea01ca4.png)
I then save and publish
Then I reload the back office, it looks like this:
![image](https://user-images.githubusercontent.com/354777/152252414-be03e4c4-2538-4094-ac53-7f9fa97896f1.png)
If I then save again (say I  edit something else on the page) then the centre information I had originally is lost

Note: this map is in an element in a nested content property

**Comment by robertjf** [comment]
@Myster thanks for reporting, this will be release in the next version within the next couple of days.

**Comment by Myster** [comment]
Also note: 
I can only access the `CenterCoordinates` via `Our.Umbraco.GMaps.Models.MapConfig` I think I should also be able to access the `PinCoordinates`
![image](https://user-images.githubusercontent.com/354777/161860439-f8d828ea-220d-4188-ac41-31f5bafc11c7.png)

**Comment by robertjf** [comment]
@Myster are you able to test the latest version (2.0.6) and see if your issue is resolved or not?

**Comment by Myster** [comment]
Hi @robertjf 
Unfortunately, I still have the same/similar issue.
The parts I see contributing to the issue are:
1) saving the content in admin should persist BOTH the **pinCoordinates** and **centerCoordinates**.
2) the front end has no access to the **pinCoordintes**

In the back office, I see the pin coordinates persist correctly, but the centre coordinates always reset to be the same as the pin when I load the back office. However, IIRC the difference is that v2.0.6 seems to not update any coordinates unless I interact with the map (drag or re-search for the address)

Persisting or not saving based on interaction would be a red-herring if both coordinates persisted. (I'm not sure if that's what you were trying to address or not)

Thanks for your work on this plugin. :-)

**Comment by robertjf** [comment]
@Myster `pinCoordinates` is represented by the `Address.Coordinates` object.  The `centerCoordinates` should be represented by the `MapConfig.CenterCoordinates` property if the map is moved, for example (taken from the example projects):

```cshtml
    <h3>Address</h3>
    <ul>
        <li>
            Coordinates (@Model.SingleMap.Address.Coordinates):
            <ul>
                <li>Latitude: @Model.SingleMap.Address.Coordinates.Latitude</li>
                <li>Longitude: @Model.SingleMap.Address.Coordinates.Longitude</li>
            </ul>
        </li>
        <li>FullAddress: @Model.SingleMap.Address.FullAddress</li>
        <li>StreetNumber: @Model.SingleMap.Address.StreetNumber</li>
        <li>Street: @Model.SingleMap.Address.Street</li>
        <li>City: @Model.SingleMap.Address.City</li>
        <li>State: @Model.SingleMap.Address.State</li>
        <li>PostalCode: @Model.SingleMap.Address.PostalCode</li>
        <li>Country: @Model.SingleMap.Address.Country</li>
    </ul>

    <h3>MapConfig</h3>
    <ul>
        <li>ApiKey: @Model.SingleMap.MapConfig.ApiKey</li>
        <li>Zoom: @Model.SingleMap.MapConfig.Zoom</li>
        <li>
            CenterCoordinates @(Model.SingleMap.MapConfig.CenterCoordinates):
            @if (Model.SingleMap.MapConfig.CenterCoordinates != null)
            {
            <ul>
                <li>Latitude: @Model.SingleMap.MapConfig.CenterCoordinates.Latitude</li>
                <li>Longitude: @Model.SingleMap.MapConfig.CenterCoordinates.Longitude</li>
            </ul>
            } else {
                <span><b>Center Coordinates IS NULL</b></span>
            }
        </li>
        <li>MapType: @Model.SingleMap.MapConfig.MapType</li>
    </ul>

```

**Comment by robertjf** [comment]
it sounds like the Umbraco Backoffice rendering of the map is ignoring `CenterCoordinates` even though they are persisted when you save the content node.  I'll take a look.

**Comment by robertjf** [comment]
@Myster I found the issue, will be fixed in release 2.0.7 in the next couple of days :)

**Comment by Myster** [comment]
Awesome, thanks @robertjf
